portal/_static/custom.css

```
@@ -1,3 +1,25 @@
+main.banner-main #project-pythia {
+  padding-top: 1rem;
+  padding-bottom: 1rem;
+}
+
+main.banner-main #project-pythia p {
+  font-size: 1.6rem;  /* default: 1.25rem */
```

Since you are trying to have a "subheading" to this section, I might actually increase the font size of the subheading (see below) and decrease the main paragraph size slightly:

```suggestion
  font-size: 1.4rem;  /* default: 1.25rem */
```